### PR TITLE
Image refresh for fedora-26

### PR DESCRIPTION
**Pull request by cockpituous**
Image refresh for fedora-26
 * [x] image-refresh fedora-26

### CODE REVIEW

**Comment by martinpitt** [comment]
@mvollmer, does this fsck failure ring a bell for you? https://fedorapeople.org/groups/cockpit/logs/pull-8529-20180207-031414-5dd443cb-verify-fedora-26/log.html#123 (the others appear to be flakes)

I don't remember seeing it before, retrying to compare.

**Comment by martinpitt** [comment]
Apparently the NTFS fsck failure is persistent, so this smells like a regression on Fedora 26.

**Comment by martinpitt** [comment]
I pushed a fix. @mvollmer, can you please review?

File: test/verify/check-storage-resize [hunk 1]
```diff
@@ -102,6 +102,7 @@ class TestStorage(StorageCase):
                          can_grow=True, grow_needs_unmount=False)
 
     @skipImage("No NTFS support installed", "centos-7", "rhel-7-5", "rhel-7-4", "rhel-7")
+    @skipImage("ntfsck broken", "fedora-26")
```
Comment (mvollmer) [inline test/verify/check-storage-resize]: Hmm, I doubt it is this simple, unfortunately.

The code calls `lvresize -r`, but it should never do that for NTFS.  I'll check more closely.

**Comment by mvollmer** [comment]
The logic in Cockpit regarding unsupported vs unrecognized was botched.  I opened #8590 with a fix.

**Comment by mvollmer** [comment]
#8590 also includes the new fedora-26 image.

**Pull request closed without merging**